migaloo/chain.json

```diff
@@ -174,6 +178,10 @@
       }
     ],
     "rest": [
+      {
+        "address": "https://migaloo-rest.polkachu.com:443",
```

Comment:
This doesn't work

Reply:
Your right for some reason its API and not rest. Not sure why I was thinking of rest. Fixed now